packages/models-library/src/models_library/basic_regex.py

```diff
@@ -45,7 +46,9 @@
 )
 
 # Storage basic file ID
-SIMCORE_S3_FILE_ID_RE = rf"^(api|({UUID_RE_BASE}))\/({UUID_RE_BASE})\/(.+)$"
+SIMCORE_S3_FILE_ID_RE = (
+    rf"^(api\/{UUID_RE_BASE}|exports\/\d+|{UUID_RE_BASE}\/{UUID_RE_BASE})\/(.+)$"
```

Comment:
TIP: might be easier to read with a constant ?
Would lead to something like
```
rf`^{API_PREFIX_RE}|{EXPORTS_USER_RE}|\/{UUID_RE_BASE}\/{UUID_RE_BASE}`\/(.+)$ 
```
?